packages/visual-dev/src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useState } from "react";
+
+import * as Styles from './Styles'
+
+interface TooltipProps {
+	text?: string;
+	direction?: 'top' | 'left' | 'bottom' | 'right'
+	children?: React.ReactNode
+}
+
+const TooltipDiv = styled.div`
+	${Styles.base}
+`;
+
+const TooltipTip = styled.div`
+	${Styles.tooltip}
+`;
+
+export const Tooltip: React.FC<TooltipProps> = ({
+	text = "",
+	direction = "top",
+	children
+}) => {
+	let delay: ReturnType<typeof setTimeout>;
+	const [active, setActive] = useState(false);
+
+	const showTooltip = () => {
+		delay = setTimeout(() => {
+			setActive(true);
+		}, 250);
```

Comment:
Having a delay on the tooltip without any animation feels weird when hovering. Generally, tooltips show up pretty much instantly once the user has hovered. Please lower the delay to something less noticeable or remove delay completely.

Reply:
I experimented with using no delay initially but found it to feel unnatural. Tooltips typically do have a bit of delay before showing on screen. You can see this when hovering over chrome tabs, photoshop tools, or even the windows start menu.

_Tooltips ... are other examples of common elements with hover interactions that can prove annoying to users if triggered too sensitively._
https://www.nngroup.com/articles/timing-exposing-content/

I'm pushing a change to make delay an optional prop with 0ms as its default value but this should allow it to be set to something else in the cases its needed.

Reply:
Having the delay as a prop makes a ton of sense